plugin/fern.vim

```diff
@@ -7,12 +7,12 @@ let g:fern_loaded = 1 " Obsolete: For backward compatibility
 command! -bar -nargs=*
       \ -complete=customlist,fern#internal#command#fern#complete
       \ Fern
-      \ call fern#internal#command#fern#command(<q-mods>, [<f-args>])
+      \ call fern#internal#command#fern#command(<q-mods>, <q-args>)
```

Comment:
I'm planning to change the entire behavior of args handling because of https://github.com/lambdalisue/fern.vim/issues/246 thus please don't touch these in this PR.